testsrc/main/mondrian/test/NativeSetEvaluationTest.java

```diff
@@ -1587,6 +1587,27 @@ public void testNativeSetsCacheClearing() {
                 getTestContext(), mdx, patterns, false, false, false);
         }
     }
+
+    /**
+    * supposes the results of native and non-native evaluations are equal.
+    *
+    * @see <a href="http://jira.pentaho.com/browse/MONDRIAN-1803">Jira issue</a>
+    */
+    public void testNativeFilterWithLargeAggSetInSlicer() {
+      // enable fallback
+      propSaver.set(
+          MondrianProperties.instance()
+              .AlertNativeEvaluationUnsupported, "ERROR");
+      final String query = "with member customers.agg as "
```

Comment:
If set to ERROR, the MDX should *fail*.  The fallback should only be used if the property is WARN.

The idea of the property is that users who need the MDX to be nativized will prefer failure over attempting to run a query which may bog down the system, so ERROR should abort.